app/ui/src/app/integrations/edit-page/action-select/action-select.component.ts

```diff
@@ -102,22 +103,35 @@ export class IntegrationsSelectActionComponent extends FlowPage
   }
 
   ngOnInit() {
-    this.actions = this.connector
-      .filter(connector => connector !== undefined)
-      .map(connector => connector.actions);
+    // set a local var for current step
```

Comment:
Do we really need to add a comment like this? Code is (and always should be) self-explanatory ;)